src/main/java/ch/jalu/configme/resource/YamlFileResource.java

```diff
@@ -278,21 +149,25 @@ protected void onWriteComplete() {
         DumperOptions options = new DumperOptions();
         options.setDefaultFlowStyle(DumperOptions.FlowStyle.BLOCK);
         options.setAllowUnicode(true);
+        options.setProcessComments(true);
+        options.setIndent(indentationSize);
```

Comment:
Maybe mention this in breaking changes as well, as I know that we override this method in AuthMe at least and might want to add these new directives, too.